src/test/java/org/springframework/data/couchbase/repository/ReactivePlace.java

```diff
@@ -0,0 +1,38 @@
+package org.springframework.data.couchbase.repository;
+
+import com.couchbase.client.java.repository.annotation.Field;
+import com.couchbase.client.java.repository.annotation.Id;
+import org.springframework.data.couchbase.core.mapping.id.GeneratedValue;
+import org.springframework.data.couchbase.core.mapping.id.GenerationStrategy;
+
+/** author
+ * David Kelly
```

Comment:
Did you mean for this to be a Javadoc "author" tag?

    /**
      * @author David Kelly
      */

Reply:
Doh!  Thx for catching that!